### PR TITLE
Portfolio: add a contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by Benawi**
## Implementation of contact form and footer Sections
# Features Added:
- Contact form Added  to the portfolio 
- The footer  section adde to the portfolio 
- data  will be Collect  using an API [Formspree service](https://formspree.io/html).
- The button interactions Implement  (hover)
-  The following form validation added to the contact form
      - Name field accepts up to 30 chars.
      - Email field accepts only emails.
      - Message field accepts up to 500 chars.
      - All fields have placeholder text.
      - All fields have the correct type.
      - A user cannot submit the form if one or more of the fields is empty.
  - I try to stick the implimantaion to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -323,24 +334,77 @@ <h3 class="custom-tags-font default-custom-font about-tag-color">Selenium</h3>
...
       </ul>
     </section>
-    
-  </body>
+    <!-- Add by charles Andrew- code Partner  -->
+    <!-- conatct form section start  -->
+    <section id="contact" class="section-contact-container flex-container-column">
+      <h2 class="legend-font-desktop primary-text-custom-font">
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+      </h2>
```
Comment (gemmen29) [inline index.html]: - [ ] Kindly remove this text as it's existed twice to match the Figma design.
image

File: index.html [hunk 2]
```diff
@@ -323,24 +334,77 @@ <h3 class="custom-tags-font default-custom-font about-tag-color">Selenium</h3>
...
+        <li class="headline-item">
+          <a class="github" href="https://github.com/Benawi" title="Contact-me: Github link"><img
+              src="./css/img/headline/icons/github.png" alt="My github"></a>
+        </li>
+        <li class="headline-item">
+          <a class="angellist" href="https://venture.angellist.com/v/login" title="Contact-me: AngelList link"><img
+              src="./css/img/headline/icons/angellist.png" alt="My angellist"></a>
+        </li>
+      </ul>
+    </footer>
```
Comment (gemmen29) [inline index.html]: - To match the Figma design:
   - [ ] Kindly add the small background on the right. 
   - [ ] Kindly add your email between the `Get in touch` button and social media icons.
   - [ ] Please center the social media icons on the middle.

| Your design | Figma design |
| --- | --- |
| image | image |

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Benawi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Benawi** [comment]
## CHANGES  ♻️
-  Email address added between the Get in touch button and social media icons.
- The social media icons set to the center
- small background added - responiseve
- The repeated text disabled for mobile version
image

**Review by gemmen29 — APPROVED** [review]
Hi @Benawi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**